test/uk/gov/hmrc/nationalinsurancerecord/controllers/NationalInsuranceRecordControllerSpec.scala

```diff
@@ -461,269 +463,244 @@ class NationalInsuranceRecordControllerSpec extends NationalInsuranceRecordUnitS
+        }
 
-      "have an Int called classTwoCredits that is 12" in {
-        (json \ "classTwoCredits").as[Int] shouldBe 12
-      }
+        "have a boolean called qualifying that is true" in {
+          (json \ "qualifying").as[Boolean] shouldBe true
+        }
 
-      "have an Int called classThreeCredits that is 0" in {
-        (json \ "classThreeCredits").as[Int] shouldBe 0
-      }
+        "have a big decimal called classOneContributions that is 1149.98" in {
+          (json \ "classOneContributions").as[BigDecimal] shouldBe 1149.98
+        }
 
-      "have an Int called otherCredits that is 10" in {
-        (json \ "otherCredits").as[Int] shouldBe 10
-      }
+        "have an Int called classTwoCredits that is 0" in {
+          (json \ "classTwoCredits").as[Int] shouldBe 0
+        }
 
-      "have an Int called classThreePayable that is 325.14" in {
-        (json \ "classThreePayable").as[BigDecimal] shouldBe 325.14
-      }
+        "have an Int called classThreeCredits that is 0" in {
+          (json \ "classThreeCredits").as[Int] shouldBe 0
+        }
 
-      "have a nullable local date called classThreePayableBy that is 5/4/2019" in {
-        (json \ "classThreePayableBy").as[LocalDate] shouldBe new LocalDate(2019, 4, 5)
-      }
+        "have an Int called otherCredits that is 0" in {
+          (json \ "otherCredits").as[Int] shouldBe 0
+        }
 
-      "have a nullable local date called classThreePayableByPenalty that is 5/4/2023" in {
-        (json \ "classThreePayableByPenalty").as[LocalDate] shouldBe new LocalDate(2023, 4, 5)
-      }
+        "have an Int called classThreePayable that is 0" in {
+          (json \ "classThreePayable").as[BigDecimal] shouldBe 0
+        }
 
-      "have a Boolean called payable that is true" in {
-        (json \ "payable").as[Boolean] shouldBe true
-      }
+        "have a nullable local date called classThreePayableBy that is null" in {
+          (json \ "classThreePayableBy") shouldBe JsDefined(JsNull)
+        }
 
-      "have a Boolean called underInvestigation that is false" in {
-        (json \ "underInvestigation").as[Boolean] shouldBe false
-      }
+        "have a nullable local date called classThreePayableByPenalty that is null" in {
+          (json \ "classThreePayableByPenalty") shouldBe JsDefined(JsNull)
+        }
 
-      "have a link to itself" in {
-        ((json \ "_links" \ "self") \ "href" ).as[String] shouldBe s"/test/ni/$testNino/taxyear/2009-10"
-      }
-    }
+        "have a Boolean called payable that is false" in {
+          (json \ "payable").as[Boolean] shouldBe false
+        }
 
-  }
+        "have a Boolean called underInvestigation that is false" in {
+          (json \ "underInvestigation").as[Boolean] shouldBe false
+        }
+
+        "have a link to itself" in {
+          ((json \ "_links" \ "self") \ "href" ).as[String] shouldBe s"/national-insurance-record/ni/$nino/taxyear/2010-11"
+        }
+      }
+  //
```

Comment:
check these tests, some seem to be duplicated while others have been removed and probably are still needed